smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -294,4 +297,33 @@ contract BridgeBank is BankStorage,
     * Don't need to do anything to handle these tokens
     */
     function tokenFallback(address _from, uint _value, bytes memory _data) public {}
+
+
+    function setRowanTokens(ERC20Burnable erowan, ERC20 rowan) public onlyOperator {
+        oldERowan = erowan;
+        newRowan = rowan;
+        emit RowanMigrationAddressSet(address(erowan), address(rowan));
+    }
+
+    /**
+    * @dev executes the migration from eRowan to Rowan. Users need to give allowance to this contract to burn eRowan before executing
```

Comment:
@banshee Could you please remove this migration code? My understanding is that we do not want to enable this transition with the IBC release.